ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -709,182 +708,186 @@ void bind_unary_composite(py::module& module, const unary_operation_t& operation
             py::arg("memory_config") = std::nullopt});
 }
 
+//OpHandler_1int
 template <typename unary_operation_t>
-void bind_unary_composite_scale_shift(py::module& module, const unary_operation_t& operation) {
+void bind_unary_composite_int_with_default(py::module& module, const unary_operation_t& operation, const std::string& parameter_name_a, const std::string& parameter_a_doc, int32_t parameter_a_value, const std::string& description) {
     auto doc = fmt::format(
```

Comment:
Why rename this function?

Reply:
to make a common structure  for all ops which have one tensor and 2 float values, because some ops have own keyword param.